visualization/app/codeCharta/ui/metricChooser/metricChooser.component.html

```diff
@@ -3,6 +3,7 @@
 	[value]="selectedMetricName"
 	(selectionChange)="handleMetricChanged($event.value)"
 	(openedChange)="handleOpenedChanged($event)"
+	[disabled]="disabled"
```

Comment:
Are all three metric chooser disabled by this?

Reply:
No! Only the metric chooser can be disabled because it get's an input property. The other choosers don't get this specific property and the default is false. Maybe I call this property 'isDisabled'.

Reply:
Ah I got it. I was counfused because the variable "disabled" has the same name as the native HTML Attribute "disabled". So I thought this is a static value, that disables all metricChoosers by default.